src/sema/statements.rs

```diff
@@ -234,6 +232,10 @@ pub fn resolve_function_body(
                 symtable.returns.push(pos);
             }
         } else {
+            if ret.ty.is_contract_storage() {
+                return_required = true;
+            }
```

Comment:
```suggestion
                return_required |= ret.ty.is_contract_storage();
```

Reply:
Why change this? What's the difference?

Reply:
It is just for shortness. Feel free to leave your code as it is.